game/views.py

```diff
@@ -102,6 +102,7 @@ class Gacha(APIView):
     softpitychance = dict()
     itemLookup = dict()
     itemChanceLookup = dict()
+    permission_classes = [AllowAny]
```

Comment:
Whats the use of this?

Reply:
It specifies that user doesn't need to be authenticated to talk to this API. You can look at the views above to see AllowAny or IsAuthenticated being permission class